merch-dropper/src/components/ProductDisplayDomain.js

```diff
@@ -13,7 +13,8 @@ const ProductDisplayDomain = ({ products, addToCart, match, location }) => {
   const [shirts, setShirts] = useState([]);
   let storeID = 0
   const { domain_name } = useParams();
-  localStorage.setItem("domain_name", domain_name)
+  console.log(domain_name)
+  localStorage.setItem("domain_name", domain_name.trim())
```

Comment:
Doesn't trim only take from front and end, if you are wanting to remove spaced in between, I think you need to use replace